### PR TITLE
Update to zen v3.3.0, version 1.0.9

### PR DESCRIPTION
**Pull request by yurikoinaba**

### CODE REVIEW

**Review by zain-cheng — APPROVED** [review]

File: docs/Release_1.0.9.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+It requires a 64-bit Windows 7 or later version to run.
+
+1. Download the Wallet EXE file
+[HorizenDesktopGUIWallet-1.0.9.exe](https://github.com/HorizenOfficial/zencash-swing-wallet-ui/releases/download/1.0.9/HorizenDesktopGUIWallet-1.0.9.exe).
+
+2. Security check: You may decide to run a virus scan on it, before proceeding... In addition using a tool 
+such as [http://quickhash-gui.org/](http://quickhash-gui.org/) you may calculate the its SHA256 checksum. The 
+result should be:
+```
+4fc4d45df0fd731c787bba18ebac96d97d99cfa56751a0130219276a4bab484d  HorizenDesktopGUIWallet-1.0.9.exe
```
Comment (cronicc) [inline docs/Release_1.0.9.md:34]: Note: Checksums need to be updated after the release in another commit

File: docs/Release_1.0.9.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+It requires Mac OS Sierra/High Sierra/Mojave.
+
+1. Download the Wallet image file
+[HorizenDesktopGUIWallet-1.0.9.dmg](https://github.com/HorizenOfficial/zencash-swing-wallet-ui/releases/download/1.0.9/HorizenDesktopGUIWallet-1.0.9.dmg).
+
+2. Security check: You may decide to run a virus scan on it before proceeding... In addition using a tool
+such as [http://quickhash-gui.org/](http://quickhash-gui.org/) you may calculate the its SHA256 checksum. The
+result should be:
+```
+93e2c2ffd3ab2bb45f8aaf84ade9858b2e7a49ad29dfa86ebef3f12fd0d7b830  HorizenDesktopGUIWallet-1.0.9.dmg
```
Comment (cronicc) [inline docs/Release_1.0.9.md:58]: Note: Checksums need to be updated after the release in another commit

**Review by cronicc — APPROVED** [review]
